llvm/lib/TargetParser/TargetParser.cpp

```diff
@@ -156,6 +157,8 @@ StringRef llvm::AMDGPU::getArchFamilyNameAMDGCN(GPUKind AK) {
   switch (AK) {
   case AMDGPU::GK_GFX9_GENERIC:
     return "gfx9";
+  case AMDGPU::GK_GFX9_4_GENERIC:
+    return "gfx9";
```

Comment:
I suppose the family name is still `gfx9` here instead of `gfx94` right?

Reply:
What is this used for? I'm not sure how this is useful

Reply:
Define those macros like `__GFX9__`

Reply:
I guess it would still be gfx9 (not that I think these type of macros are useful)